src/Distance/Distance.php

```diff
@@ -146,10 +146,10 @@ public function haversine()
         $dLat = $latB - $latA;
         $dLon = $lngB - $lngA;
 
-        $a = sin($dLat / 2) * sin($dLat / 2) + cos($latA) * cos($latB) * sin($dLon / 2) * sin($dLon / 2);
-        $c = 2 * atan2(sqrt($a), sqrt(1 - $a));
+        $a = (sin($dLat / 2) ** 2) + cos($latA) * cos($latB) * (sin($dLon / 2) ** 2);
+        $c = 2 * asin(sqrt($a));
```

Comment:
Could you help me understan why we need this change

Reply:
The Haversine distance calculation is optimized now, a little bit faster :)